test/unit/common/Reducers.test.js

```diff
@@ -655,4 +655,13 @@ describe("Reducers", () => {
       assert.equal(state.pocketCta.useCta, data.use_cta);
     });
   });
+  describe("Layout", () => {
+    it("should return INITIAL_STATE by default", () => {
+      assert.equal(Layout(undefined, {type: "some_action"}), INITIAL_STATE.Layout);
+    });
+    it("should set layout with CONTENT_LAYOUT", () => {
```

Comment:
Maybe "it should set layout.type to CONTENT_LAYOUT"?

Reply:
How about "should set layout data with layout.type CONTENT_LAYOUT"?

Reply:
Sure